docs/sim/cli.rst

```diff
@@ -0,0 +1,236 @@
+===
+scl
+===
+
+Usage: scl [OPTIONS] COMMAND [ARGS]...
+
+  The SMARTS command line interface. Use --help with each command for further
+  information.
+
+Options:
+  --help  Show this message and exit.
+
+Commands:
+  document-help  Write SCL help information to docs.
```

Comment:
`document-help` command is not present. 

This entire file can be removed, as this file is going to be auto generated and overwritten by the `ci-auto-commit-linux.yml`.